gradle/verification-metadata.xml

```diff
@@ -3049,6 +3054,11 @@
             <sha256 value="a24485517596ae1003dcf2329c044a2a861e5c25d4476a695ccaacf560c74d1a" origin="Generated by Gradle because artifact wasn't signed"/>
          </artifact>
       </component>
+      <component group="org.owasp.encoder" name="encoder" version="1.2.2">
```

Comment:
Why do we have this now?

Reply:
Transitive dependency of flightrecorder: https://ge.gradle.org/s/5wkujgzpxlfvi/dependencies?dependencies=owasp&expandAll&focusedDependency=WzY5LDE3LDY1NjYsWzAsMCxbMF1dXQ&focusedDependencyView=versions